src/app/Mappings/Storage/mocks/storage_mappings.mock.ts

```diff
@@ -0,0 +1,132 @@
+import { IStorageMapping, INetworkMapping, MappingType } from '../../types';
+import { ProviderType } from '@app/common/constants';
+import { MappingSource, MappingTarget } from '@app/Mappings/types';
+import { NetworkType } from '@app/Providers/types';
+
+const storageMapping1: IStorageMapping = {
```

Comment:
This filename is `storage_mappings.mock.ts`, it contains both storage and network mappings, so maybe we should call it `mappings.mock.ts`